pkg/operator/render.go

```diff
@@ -83,6 +83,8 @@ func platformFromInstallConfig(ic installertypes.InstallConfig) string {
 		return "openstack"
 	case ic.Libvirt != nil:
 		return "libvirt"
+	case ic.None != nil:
```

Comment:
Why not replace `platformFromInstallConfig(config)` with `config.Platform.Name()`?  Or use `config.Platform.Name()` inside `platformFromInstallConfig` and raise errors on empty-string results?

Reply:
I'd rather handle this in a different PR, I'm not sure if other components are relying on the function - and unblocking BYOR ASAP is the priority